Content.Server/Revolutionary/Components/Exiled.cs

```diff
@@ -0,0 +1,12 @@
+using Content.Server.Flash;
+using Content.Server.GameTicking.Rules;
+using Robust.Shared.GameStates;
+
+namespace Content.Server.Revolutionary.Components;
+/// <summary>
+/// Given to heads that decide to run away from station during Revs.
+/// </summary>
+[RegisterComponent, Access(typeof(RevolutionaryRuleSystem))]
+public sealed class ExiledComponent : Component
```

Comment:
i feel like this is a really vague name for what is a pretty specific idea. It's fine if the name is a bit long and clunky, so long as it is a better description of what it does.